infrastructure/module/database/namespace.tf

```diff
@@ -0,0 +1,4 @@
+module "namespace" {
```

Comment:
This is not needed, you can create namespace via helm chart with:

**create_namespace = true**